proxy/src/control/destination/mod.rs

```diff
@@ -228,10 +248,38 @@ impl Responder {
 
 // ===== impl Metadata =====
 
+impl Default for Metadata {
+    fn default() -> Self {
+        Self::no_metadata()
+    }
+}
```

Comment:
Sorry, i didn't notice this before. Why did we add this now? It seems like it's not used and I would prefer to be more explicit about intetionally not having any metadata.

Reply:
At one point there were some uses of `unwrap_or_default`, but that's no longer the case. I just checked, and this can safely be removed.